lib/question.js

```diff
@@ -518,9 +518,8 @@ module.exports = {
                 });
             },
             (callback) => {
-                if (question.grading_method == 'Internal') {
-                    // for Internal grading we call the grading code
-                    questionModule.grade(submission, variant, question, course, (err, ret_courseIssues, ret_data) => {
+                // We always want to call the internet grader in case it can help grade external / manual questions with internal parts
```

Comment:
```suggestion
                // We always want to call the internal grader in case it can help grade external / manual questions with internal parts
```